src/fromevent.js

```diff
@@ -39,13 +42,21 @@ var findHandlerMethods = function(target) {
   throw new Error("No suitable event methods in " + target);
 };
 
-Bacon.fromEventTarget = function(target, eventName, eventTransformer) {
+Bacon.fromEventTarget = function(target, eventName, ...args) {
+  let optionsOrUseCapture, eventTransformer;
+
+  if (_.isFunction(args[0])) {
+    [eventTransformer] = args;
+  } else {
+    [optionsOrUseCapture, eventTransformer] = args;
+  }
+
   var [sub, unsub] = findHandlerMethods(target);
   var desc = new Bacon.Desc(Bacon, "fromEvent", [target, eventName]);
   return withDesc(desc, Bacon.fromBinder(function(handler) {
-    sub.call(target, eventName, handler);
+    sub.call(target, eventName, handler, optionsOrUseCapture);
     return function() {
-      return unsub.call(target, eventName, handler);
+      return unsub.call(target, eventName, handler, optionsOrUseCapture);
```

Comment:
I got the naming from this page: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

In this documentation, the 3rd parameter is called either `options` or `useCapture`.

Reply:
hrmm, but it's headed to a bacon function, and it's not clear that it would be passed to the listener.

`listenerOptionsOrUseCapture` kind of implies this is headed to the listener to me.

Reply:
Fair enough.